Cargo.toml

```diff
@@ -20,7 +20,7 @@ exclude = ["python"]
 resolver = "2"
 
 [workspace.package]
-version = "0.13.1"
+version = "0.14.0"
```

Comment:
Are you also bumping the lance version?

Reply:
Doing to to pass the breaking change policy check.